README.adoc

```diff
@@ -1,18 +1,62 @@
+[#trackit-logo]
+image::https://s3-us-west-2.amazonaws.com/trackit-public-artifacts/github-page/logo.png[]
```

Comment:
I think the logo should be smaller (at least 50%) ; this is a bit burlesque.

Reply:
It's a detail for now. We can change it later on.